charts/artifact-hub/values.yaml

```diff
@@ -313,6 +319,8 @@ scanner:
       # Scanner image repository (without the tag)
       repository: artifacthub/scanner
     securityContext: {}
+    containerSecurityContext: {}
+    serviceAccountName: default
```

Comment:
These default values don't seem to be used (maybe you meant to add them to the tracker block?).